Getting-Started.md

```diff
@@ -37,6 +37,87 @@ You will need to install ElasticSearch (6.1.0), Kibiter (6.1.4) and a MySQL/Mari
 You will have to install the GrimoireLab components listed above, and use the following docker-compose to have ElasticSearch, Kibiter and MariaDB.
 Note that you can omit the `mariadb` section in case you have MySQL/MariaDB already installed in your system.
 
+***
+
+#### Docker installation
```

Comment:
Can you move the Docker/Docker-compose installation to a different PR?

Since the section "source code and docker" is becoming too large, I would propose to:
- move your new content to a section called "Install Docker and Docker compose"
- the section "source code and docker" should be renamed "source code and docker compose". It should include a link to the new section (e.g., "Install Docker and Docker compose"). For instance:
```
You will have to install the GrimoireLab components listed above, and use the following docker-compose to have ElasticSearch, Kibiter and MariaDB. Note that you can omit the mariadb section in case you have MySQL/MariaDB already installed in your system.

See section .. to install (docker and docker compose](link)
``` 
- the section "Only Docker" should include a link to the new section "Install Docker and Docker compose"

WDYT?

Reply:
Okay..Will remove the installation parts for now, we can think of some better option later.
Thanks!